packages/graph/round/docs/HandlingUpgrades.md

```diff
@@ -17,7 +17,7 @@ Note: Anytime an enitity is updated with a new property. Increment the version p
 Let's assume we are updating `QuadraticFundingVotingStrategyImplementation`
 
 **PreRequisites**
-1. Ensure the `QuadraticFundingVotingStrategyImplementation` contract changes are done and abi is genered from `packages/contract`
+1. Ensure the `QuadraticFundingVotingStrategyImplementation` contract changes are done and abi is genered from `contract`
```

Comment:
Maybe add a link to [`contract`](https://github.com/Allo-Protocol/contracts) since it's now in a separate org?